### PR TITLE
Removes Geometry Proxy and setUpdateCallback function

### PR DESCRIPTION
**Pull request by jtorresfabra**

### CODE REVIEW

**Comment by stephomi** [comment]
Maybe now we can simply keep `_primitives` and `_attributes`.

**Comment by jtorresfabra** [comment]
Fixed.

**Comment by stephomi** [comment]
+1

**Comment by cedricpinson** [comment]
@jtorresfabra needs to check the old deprecated parse scenegraph if we dont break something https://github.com/cedricpinson/osgjs/blob/performance-test/sources/osgDB/readerParser.js#L186
I wanted to remove the code but I believe we still have old files

**Comment by jtorresfabra** [comment]
oups, missed these. Fixed now.

File: sources/osgUtil/IntersectionVisitor.js [hunk 1]
```diff
@@ -156,7 +156,7 @@ IntersectionVisitor.prototype = MACROUTILS.objectInherit( NodeVisitor.prototype,
 
     applyNode: function ( node ) {
         if ( !this.enter( node ) ) return;
-        if ( node.primitives ) {
+        if ( node.getPrimitiveSetList ) {
```
Comment (cedricpinson) [inline sources/osgUtil/IntersectionVisitor.js]: it's probably better to use 'instanceof Geometry' it makes more sense to read

Comment (cedricpinson) [inline sources/osgUtil/IntersectionVisitor.js]: my bad it's faster to check property https://jsperf.com/instanceof-performance/25

**Pull request merged**